hw/ip/otbn/dv/model/iss_wrapper.cc

```diff
@@ -536,6 +536,12 @@ void ISSWrapper::send_err_escalation(uint32_t err_val, bool lock_immediately) {
   run_command(oss.str(), nullptr);
 }
 
+void ISSWrapper::send_stall_request() {
+  std::ostringstream oss;
+  oss << "send_stall_request\n";
+  run_command(oss.str(), nullptr);
```

Comment:
I think you can just pass a string literal to `run_command` and not mess around with an `ostringstream` here. (There are no arguments)